tendermint/src/rpc/endpoint/abci_info.rs

```diff
@@ -27,41 +27,58 @@ impl rpc::Response for Response {}
 
 /// ABCI information
 #[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(default)]
 pub struct AbciInfo {
     /// Name of the application
     pub data: String,
 
     /// Version
+    #[serde(skip_serializing_if = "Option::is_none")]
     pub version: Option<String>,
 
     /// Last block height
-    pub last_block_height: block::Height,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub last_block_height: Option<block::Height>,
 
     /// Last app hash for the block
     #[serde(
         serialize_with = "serialize_app_hash",
-        deserialize_with = "parse_app_hash"
+        deserialize_with = "parse_app_hash",
+        skip_serializing_if = "Option::is_none"
     )]
-    pub last_block_app_hash: Hash,
+    pub last_block_app_hash: Option<Hash>,
```

Comment:
Similar to above: This can only be empty on genesis, right? Maybe add a comment.

Reply:
Nope, abci_info doesn't print this over RPC.